src/base-element.d.ts

```diff
@@ -107,9 +107,6 @@ declare namespace AMP {
   ) => void;
 }
 
-// TODO(samouri): Move the types below to better locations.
-declare type AmpElement = HTMLElement;
-
 declare type Layout =
```

Comment:
I think `LayoutSize` and `LayoutRect` can be imported from `#core/dom/layout`

Reply:
Sick. I'll do that.

Reply:
Ok that didn't work. Cannot import a module from an ambient context :(

Reply:
You should be able to import it from within the namespace, though, which is where the types are used (or just import inline on lines 27/28